ogr/services/gitlab/release.py

```diff
@@ -1,36 +1,89 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
-from typing import Optional
+from gitlab.v4.objects import ProjectRelease as _GitlabRelease
+
+import datetime
+from typing import Optional, List
 
 from ogr.abstract import Release, GitTag
 from ogr.services import gitlab as ogr_gitlab
 from ogr.exceptions import OperationNotSupported
 
 
 class GitlabRelease(Release):
+    _raw_release: _GitlabRelease
     project: "ogr_gitlab.GitlabProject"
 
-    def __init__(
-        self,
-        tag_name: str,
-        url: Optional[str],
-        created_at: str,
-        tarball_url: str,
-        git_tag: GitTag,
-        project: "ogr_gitlab.GitlabProject",
-        raw_release,
-    ) -> None:
-        super().__init__(tag_name, url, created_at, tarball_url, git_tag, project)
-        self.raw_release = raw_release
-
     @property
     def title(self):
-        return self.raw_release.name
+        return self._raw_release.name
 
     @property
     def body(self):
-        return self.raw_release.description
+        return self._raw_release.description
+
+    @property
+    def git_tag(self) -> GitTag:
+        return self.project._git_tag_from_tag_name(self.tag_name)
+
+    @property
+    def tag_name(self) -> str:
+        return self._raw_release.tag_name
+
+    @property
+    def url(self) -> Optional[str]:
+        # no link is provided
+        return None
```

Comment:
Can't we guess/construct that URL?

Reply:
Well, now I am puzzled, cause it appears that GitLab has a specific interface for releases and you could potentially guess URL, cause it uses _tag name_ in the URL pointing to the release. Will need to have a better look at it, cause I have no clue how recent that feature is.